corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -604,6 +640,10 @@ public int deleteRecords(String namespace, String tableName, List<String> keysTo
                     continue;
                 }
                 System.out.println(numKeysDeleted + ": Deleting record with Key " + keyToDelete);
+                if (keysToDelete.size() < 5) {
```

Comment:
We are logging only if less than 5 entries are deleted, right?
Do we need to log some metadata for the other deleted entries for the audit trail?

Reply:
We discussed this - and thought only the log line `BULK DELETE {} records from tablename` should be printed since they could be using bulk delete to potentially remove millions of entries

Reply:
In that case, there will be no audit trail for bulk deletes.  Is it acceptable?

Reply:
yes the risk is if we log all the bulk deletes the file will rollover too soon and we won't be able to capture other changes.